src/test/java/ay2021s1_cs2103_w16_3/finesse/logic/parser/bookmark/AddBookmarkExpenseCommandParserTest.java

```diff
@@ -42,16 +40,6 @@ public void parse_allFieldsPresent_success() {
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + TITLE_DESC_PHONE_BILL + AMOUNT_DESC_PHONE_BILL
                 + CATEGORY_DESC_UTILITIES, new AddBookmarkExpenseCommand(expectedBookmarkExpense));
 
-        // multiple titles - last title accepted
-        assertParseSuccess(parser, TITLE_DESC_SPOTIFY_SUBSCRIPTION + TITLE_DESC_PHONE_BILL
-                        + AMOUNT_DESC_PHONE_BILL + CATEGORY_DESC_UTILITIES,
-                new AddBookmarkExpenseCommand(expectedBookmarkExpense));
-
-        // multiple amounts - last amount accepted
-        assertParseSuccess(parser, TITLE_DESC_PHONE_BILL + AMOUNT_DESC_SPOTIFY_SUBSCRIPTION
-                + AMOUNT_DESC_PHONE_BILL + CATEGORY_DESC_UTILITIES,
-                new AddBookmarkExpenseCommand(expectedBookmarkExpense));
-
```

Comment:
Is it possible to convert these into negative test cases instead? It will help us avoid extra work later on if coverage needs to be brought back up.